frontend/src/components/Sidepanel/Panels/TestingLab/TestingLab.js

```diff
@@ -80,6 +81,15 @@ const TestingLab = () => {
     return `${transitionsWithRejected.join('\n')}${(traceIdx === transitionCount) ? `\n\n` + (accepted ? 'ACCEPTED' : 'REJECTED' ) : ''}`
   }, [traceInput, simulationResult, statePrefix, traceIdx])
 
+  useEffect(() => {
+    setMultiTraceOutput(multiTraceInput.map(input => simulateFSA(graph, input)))
```

Comment:
This is unnecessary, it will be run on line 89 as well.

Reply:
Removed redundant useEffect